lib/osf-components/addon/components/file-browser/breadcrumbs/template.hbs

```diff
@@ -0,0 +1,9 @@
+<h3 local-class='Breadcrumbs' data-test-breadcrumbs>
+    {{#each @breadcrumbs as |breadcrumb index| }}
+        {{#if (eq index 0)}}
+            {{t (concat 'registries.overview.files.storage_providers.' breadcrumb.name)}}
+        {{else}}
+            <span local-class='Divider'>></span> {{breadcrumb.name}}
```

Comment:
Should we style this as something clickable and then add an action to navigate to the clicked folder?

Reply:
Looks easy enough. I'll take a look.